android/java/res/layout/fragment_onboarding_network_selection.xml

```diff
@@ -72,9 +70,7 @@
             android:layout_alignParentEnd="true"
             android:layout_below="@+id/network_search_layout"
             android:text="@string/show_testnets"
-            android:textColor="@color/wallet_text_color"
-            android:lineSpacingExtra="4sp"
-            android:textSize="14sp" />
+            android:textColor="@color/wallet_text_color" />
```

Comment:
same here

Reply:
Done in commit 👉 [`a85fb75` (#24063)](https://github.com/brave/brave-core/pull/24063/commits/a85fb75ba0e9f37da89ed68e35e9141a90eadbdf)